larpix/larpix.py

```diff
@@ -838,6 +868,31 @@ def bytes(self):
         bytes_output = padded_output.bytes
         return bytes_output[::-1]
 
+    def export(self):
+        '''Return a dict representation of this Packet.'''
+        type_map = {'00': 'test', '01': 'data', '10': 'config write',
+                '11': 'config read'}
+        d = {}
+        d['bits'] = self.bits.bin
+        d['type'] = type_map[self.packet_type.bin]
+        d['chipid'] = self.chipid
+        d['parity'] = self.parity_bit_value
+        d['valid_parity'] = self.has_valid_parity()
+        ptype = self.packet_type
+        if ptype == Packet.TEST_PACKET:
+            d['counter'] = self.test_counter
+        elif ptype == Packet.DATA_PACKET:
+            d['channel'] = self.channel_id
+            d['timestamp'] = self.timestamp
```

Comment:
Dan and I discussed timestamping a few days ago. We think the right way to do it is to force-trigger once every 20-30 seconds so that we can correlate the computer timestamp with the LArPix timestamp.

All this is to say that the timestamping is going to be a bit more complex so in classic grad student fashion I am just ignoring it for now.